spec/index.html

```diff
@@ -703,60 +711,60 @@ <h3>Examples</h3>
         </pre>
 
         <p><a href="#ca-2">Step 2</a> is called four times, with each blank node
-          (`_:e0`, `_:e1`, `_:e2`, and `_:e3`)
+          (`e0`, `e1`, `e2`, and `e3`)
           to populate <a>blank node to quads map</a>:</p>
 
         <table id="ex-ca-shared-bn-quads-map"
                class="hash-related numbered">
           <caption>Blank node to quads map for shared hashes</caption>
           <thead><tr>
-            <th><var>identifier</var></th>
+            <th><var>blank node</var></th>
             <th><var>Q</var></th>
```

Comment:
Same here, its a "blank node to quads map", so showing a "blank node" column corresponds.